cmd/tiller/release_server.go

```diff
@@ -238,13 +238,17 @@ func (s *releaseServer) InstallRelease(c ctx.Context, req *services.InstallRelea
 
 	// Render the templates
 	// TODO: Fix based on whether chart has `engine: SOMETHING` set.
-	vals, err := chartutil.CoalesceValues(req.Chart, req.Values, overrides)
+	//vals, err := chartutil.CoalesceValues(req.Chart, req.Values, overrides)
```

Comment:
I'd like to know your convention on commented out lines. 

Do you prefer: 1 - to keep them commented out for future reference or 2 - to delete them and use git history if needed?

My take is 2 but I am open to what the general guidelines say :)